superset/assets/spec/javascripts/sqllab/QuerySearch_spec.jsx

```diff
@@ -19,13 +19,7 @@ describe('QuerySearch', () => {
   const wrapper = shallow(<QuerySearch {...mockedProps} />);
 
   it('should have four Select', () => {
-    expect(wrapper.find(Select)).to.have.length(4);
-  });
-
-  it('updates userId on user selects change', () => {
-    wrapper.find('[name="select-user"]')
-      .simulate('change', { value: 1 });
-    expect(wrapper.state().userId).to.equal(1);
+    expect(wrapper.find(Select)).to.have.length(3);
```

Comment:
ok cool. we should update the `it` statement on L21 to read `should have three selects` in that case.